### DISCUSSION TITLE
What is the longest word in the English language?

### DISCUSSION THREAD

**Discussion by narayan954** [ParanoidUser/yolo#237] [Galaxy Trivia] [answered]
Hint: A mouthful lung disease

Bet you can't write that one in one try

**Comment by thenithinbalaji** [comment, marked as answer]
Pneumonoultramicroscopicsilicovolcanoconiosis

**Reply by narayan954** [reply]
are you sure, you didn't just copy-paste it? xD

**Reply by ParanoidUser** [reply]
This word is begging for spaces and commas 😁
